packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -792,6 +800,11 @@ export class Function extends FunctionBase {
         const runtimes = layer.compatibleRuntimes.map(runtime => runtime.name).join(', ');
         throw new Error(`This lambda function uses a runtime that is incompatible with this layer (${this.runtime.name} is not in [${runtimes}])`);
       }
+
+      // Currently no validations for compatible architectures since Lambda service
+      // allows layers configured one architecture to be used with a Lambda function
```

Comment:
Should this be the following?

```suggestion
      // allows layers configured with one architecture to be used with a Lambda function
```

Reply:
Great catch!